### PR TITLE
[RSDK-8688] update workflows after move

### PR DESCRIPTION
**Pull request by JohnN193**
https://viam.atlassian.net/browse/RSDK-8688

not sure what to do about GCP stuff yet, otherwise I think I covered all of the big changes

### CODE REVIEW

File: .github/workflows/pullrequest-trusted.yml [hunk 1]
```diff
@@ -8,24 +8,23 @@ concurrency:
...
     branches: [ 'main' ]
-    types: [ 'labeled' ]
 
 # To test workflow updates you need to work in a branch directly on viamrobotics/viam-cartographer
 # and tag your working branch instead of @main in any viamrobotics/viam-cartographer "uses" below.
 # Don't forget to tag back to @main before merge.
 
 jobs:
   test:
-    if: github.event.label.name == 'safe to test' ||
```
Comment (abe-winter) [inline .github/workflows/pullrequest-trusted.yml:19]: note -- you probably copied this complex `@main` stuff from a repo that accepts frequent outside contributions

as long as this repo keeps the 'Require approval for all outside collaborators' setting, you can change `viamrobotics/viam-cartographer/.github/workflows/test.yml@main` calls to `./.github/workflows/test.yml`, which makes life easier in a bunch of cases

(but up to you whether to make this change)

**Review by abe-winter — APPROVED** [review]
guessing you're planning to change `@workflow-updates` back to `@main` when ready

**Comment by abe-winter** [comment]
for GCP stuff, copy [realsense repo appimage.yml](https://github.com/viam-modules/viam-camera-realsense/blob/main/.github/workflows/appimage.yml#L47-L63) when you're ready

File: .github/workflows/pullrequest-trusted.yml (hunk 1 above)
Comment (JohnN193) [reply .github/workflows/pullrequest-trusted.yml:19]: so with this change i won't have to jump thru hoops to test a change to the test workflow?

Comment (abe-winter) [reply .github/workflows/pullrequest-trusted.yml:19]: yup exactly

**Pull request merged**